src/clue/adafruit_clue.py

```diff
@@ -250,10 +265,186 @@ def were_pressed(self):
+          from adafruit_clue import clue
+          while True:
+              print("Proximity: {}".format(clue.proximity))
+        """
+        return self.__state[CONSTANTS.CLUE_STATE.PROXIMITY]
+
+    @property
+    def gyro(self):
+        """Obtain x, y, z angular velocity values in degrees/second.
+        This example prints the values. Try moving the board to see how the printed values change.
+              print("Gyro: {:.2f} {:.2f} {:.2f}".format(*clue.gyro))
+        """
+        return (
+            self.__state[CONSTANTS.CLUE_STATE.GYRO_X],
+            self.__state[CONSTANTS.CLUE_STATE.GYRO_Y],
+            self.__state[CONSTANTS.CLUE_STATE.GYRO_Z],
+        )
+
+    @property
+    def gesture(self):
+        """A gesture code if gesture is detected. Shows ``0`` if no gesture detected.
+        ``1`` if an UP gesture is detected, ``2`` if DOWN, ``3`` if LEFT, and ``4`` if RIGHT.
+        This example prints the gesture values. Try moving your hand up, down, left or right over
+        the sensor to see the value change.
+        To use with the CLUE:
+        .. code-block:: python
+          from adafruit_clue import clue
+          while True:
+              print("Gesture: {}".format(clue.gesture))
+        """
+        gesture_mapping = {"": 0, "up": 1, "down": 2, "left": 3, "right": 4}
+        return gesture_mapping.get(self.__state[CONSTANTS.CLUE_STATE.GESTURE], 0)
+
+    @property
+    def humidity(self):
+        """The measured relative humidity in percent.
+        This example prints the value. Try breathing on the sensor to see the values change.
+        To use with the CLUE:
+        .. code-block:: python
+          from adafruit_clue import clue
+          while True:
+              print("Humidity: {:.1f}%".format(clue.humidity))
+        """
+        return self.__state[CONSTANTS.CLUE_STATE.HUMIDITY]
+
+    @property
+    def pressure(self):
+        """The barometric pressure in hectoPascals.
+        This example prints the value.
+        To use with the CLUE:
+        .. code-block:: python
+            from adafruit_clue import clue
+            print("Pressure: {:.3f}hPa".format(clue.pressure))
+        """
+        return self.__state[CONSTANTS.CLUE_STATE.PRESSURE]
+
+    @property
+    def altitude(self):
+        """The altitude in meters based on the sea level pressure at your location. You must set
+        ``sea_level_pressure`` to receive an accurate reading.
+        This example prints the value. Try moving the board vertically to see the value change.
+        .. code-block:: python
+            from adafruit_clue import clue
+            clue.sea_level_pressure = 1015
+            print("Altitude: {:.1f}m".format(clue.altitude))
+        """
+        altitude = 44330 * (
+            1.0
+            - math.pow(
+                self.__state[CONSTANTS.CLUE_STATE.PRESSURE]
+                / self.__state[CONSTANTS.CLUE_STATE.SEA_LEVEL_PRESSURE],
+                0.1903,

```

Comment:
For me, I don't think it makes sense to make them constants because we don't use them anywhere else and constants may make it more confusing? Might just be me.

Reply:
I agree with Andrea here. It's better to not have magic number, if you can find that constant name, it's better if you declare it. If it's used only here, it can be local

Reply:
Actually since they don't provide the constants name in the source code, I think it's not worth the additional effort. Linking the source code might help if it's not already done.

Reply:
Updated it to have constants :p